source/developer/bot-accounts.rst

```diff
@@ -0,0 +1,186 @@
+.. _bot-accounts:
+
+Bot Accounts
+========================================
+
+Use Bot Accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the :doc:`personal access tokens feature <personal-access-tokens>`.
+
+// TODO screenshot. Waiting for bug fix where channel header is missing from bot DMs
+
+Bot accounts are just like user accounts, except they:
+
+  - Cannot be logged into, and bypass all restrictions related to authentication, such as multi-factor authentication (MFA) and custom terms of service.
+  - Cannot be used to create other bot accounts.
+  - Do not count as a registered user and therefore do not count towards the total users for an Enterprise Edition license.
+
+Given bot accounts cannot be used to log in, they enable developers to create integrations that don’t require an email/password login or any other authentication method for their bots. This removes unnecessary attack surface caused by creating and in some cases circumventing authentication methods to create a bot account.
```

Comment:
>  or any other authentication method

They still require authentication. Not sure what this is trying to say.

Reply:
How is this

```
Bot accounts are a safer way to integrate with Mattermost through the RESTful API and Plugin API because they cannot be used to log in. They enable developers to create integrations that don’t require an email/password login, removing an attack surface caused by creating and in some cases circumventing authentication methods to create a bot account.
```

In essence I want to tell admins 'you should use bot accounts because it makes your system more secure'

Reply:
It's more secure because you don't have to manage shared logins to these accounts.

Reply:
Updated, see https://github.com/mattermost/docs/pull/2761/commits/a3533da93776e09daf86c34c78190d481726cda4